### DISCUSSION TITLE
mdSilo Desktop v0.4.1

### DISCUSSION THREAD

**Discussion by danloh** [mdSilo/mdSilo#5] [Announcements]
### Added
- Table of Contents
- WikiLink 

### Fixed
- Bug on create new note 

<hr /><em>This discussion was created from the release <a href='https://github.com/mdSilo/mdSilo/releases/tag/app-v0.4.1'>mdSilo Desktop v0.4.1</a>.</em>

**Comment by pramjana** [comment]
Is there any scope for exporting notes in the future? As admonitions are specific to mdSilo, and can't be processed by other markdown exporters? What's the correct approach for this requirement?

**Reply by danloh** [reply]
most common Markdown features are supported by other markdown editor, excerpt Math, call-out. 

Will work on the exporting note in the future.

**Reply by danloh** [reply]
if there is no support of admonitions(call-out) in other editor, i do not know any approach, sorry.

**Comment by pramjana** [comment]
Your app is fabulous, I wanted to let you know! It's super fast, and got some nice WYSIWYG features 🔥 . I got to know about tauri thanks to your app! Please keep up the great job!!
